app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,18 @@
+# class Api::V1::GreetingsController < ApplicationController
+#   # GET /greetings or /greetings.json
+#   def index
+#     @greetings = [Greeting.order('RANDOM()').first]
+#     render json: @greetings
+#   end
+# end
+
+# app/controllers/api/v1/greetings_controller.rb
+module Api
+  module V1
+    class GreetingsController < ApplicationController
+      def index
+        @greetings = Greeting.all
```

Comment:
- [ ] Hello, kindly note your API should return a random greeting from the database

```suggestion
        @random_greeting = Greeting.order('RANDOM()').first
```